app/views/welcome/index.html.haml

```diff
@@ -1,44 +1,52 @@
-.root-landing-page
-  %header.root-landing-banner-header
+%main.root-landing-page
+  %header.jumbotron.root-landing-banner-header.bg-image-bike-hero
     .container
-      .row
-        .root-landing-header-text
-          %p
-            %strong
-              = number_with_delimiter Counts.recoveries
-            bikes recovered.
-          %h1
-            <span>Bike</span> <span>Registration</span> <span>that</span> <span>Works</span>
-          %a.btn.btn-lg.btn-success.root-action-btn{ href: new_user_path }
-            Get started, it's free
-        .root-landing-signup
-          - unless current_user.present?
-            = form_for User.new do |f|
-              .form-group
-                = f.label :name, 'full name', class: 'sr-only'
-                = f.text_field :name, placeholder: 'name', class: 'form-control'
-              .row
-                .form-group.col-md-6
-                  = f.label :email, class: 'sr-only'
-                  = f.email_field :email, placeholder: 'email', class: 'form-control'
-                .form-group.col-md-6
-                  = f.label :password, class: 'sr-only'
-                  = f.password_field :password, placeholder: 'password', class: 'form-control'
+      .root-landing-header-text
+        %h2
+          Over
+          = number_with_delimiter(@recovery_count)
```

Comment:
Should be `Counts.recoveries_value` - and `@recovery_count` can be fully removed.

Reply:
`@recovery_count` is cached with a daily refresh -- let me know if you want to get rid of that

Reply:
Yeah, getting rid of it is fine. The `Counts.recoveries_value` is cached (calculated and stored in redis) via [update_counts_worker](https://github.com/bikeindex/bike_index/blob/master/app/workers/update_counts_worker.rb)